fineract-client/src/main/templates/java/retrofit2/ApiClient.mustache

```diff
@@ -0,0 +1,468 @@
+{{>licenseInfo}}
+package {{invokerPackage}};
```

Comment:
These files look like boilerplate... have we modified them? If not, is there a possibility to have them as dependency we download rather than something we check into git?

Reply:
We could leave them out; the templates are already available as a dependency... but: I thought we want at least the license info included and this is the only way to do it.

Reply:
If it's just for the license headers, then yes I would remove this, to reduce ma maintenance. If we need to improve or fix things, then we should, ideally, contribute that upstream... Remove?